packages/react-error-overlay/src/containers/RuntimeError.js

```diff
@@ -65,4 +65,5 @@ function RuntimeError({ errorRecord, launchEditorEndpoint }: Props) {
   );
 }
 
+export type { ErrorRecord };
```

Comment:
Can you please just add `export` to `type` definition itself?

Reply:
Apparently I can't.
![image](https://user-images.githubusercontent.com/5604548/30166422-ea4016d6-9415-11e7-877b-cb9a12035635.png)

Reply:
Or you probably meant:
```
export type ErrorRecord = {|
  error: Error,
  unhandledRejection: boolean,
  contextSize: number,
  stackFrames: StackFrame[],
|};
```

Reply:
Yes, that's what I meant.